diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1111,3 +1111,113 @@ impl InetOrCidr for Inet {}
 impl InetOrCidr for Cidr {}
 impl InetOrCidr for Nullable<Inet> {}
 impl InetOrCidr for Nullable<Cidr> {}
+
+/// PostgreSQL specific expression methods using the `Jsonb` datatype.
+pub trait PgJsonbExpressionMethods: Expression<SqlType = Jsonb> + Sized {
+    /// Creates a PostgreSQL `||` operator
+    ///
+    /// This operator merges two `Jsonb` objects.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # include!("../../doctest_setup.rs");
+    /// #
+    /// # table! {
+    /// #     characters {
+    /// #         id -> Integer,
+    /// #         a_b_c -> Jsonb,
+    /// #         d_e_f -> Jsonb,
+    /// #     }
+    /// # }
+    /// #
+    /// # #[cfg(feature = "serde_json")]
+    /// # fn main() {
+    /// #     run_test();
+    /// # }
+    /// #
+    /// # fn run_test() {
+    /// #     use diesel::insert_into;
+    /// #     use self::characters::dsl::*;
+    /// #     use serde_json::from_str;
+    /// #
+    /// #     let connection = connection_no_data();
+    /// #     diesel::sql_query("CREATE TABLE characters (
+    /// #         id SERIAL PRIMARY KEY,
+    /// #         a_b_c JSONB NOT NULL,
+    /// #         d_e_f JSONB NOT NULL
+    /// #     )").execute(&connection)?;
+    /// #
+    /// #     // insert into a_b_c
+    /// #     let a_b_c_data = r#"
+    /// #         {
+    /// #             "a": 1,
+    /// #             "b": 2,
+    /// #             "c": 3,
+    /// #             "letters": [
+    /// #                 "a",
+    /// #                 "b",
+    /// #                 "c",
+    /// #             ]
+    /// #         }
+    /// #     "#;
+    /// #     diesel::insert_into(a_b_c).values(from_str(a_b_c_data));
+    /// #
+    /// #     // insert into d_e_f
+    /// #     let d_e_f_data = r#"
+    /// #         {
+    /// #             "d": 1,
+    /// #             "e": 2,
+    /// #             "f": 3,
+    /// #             "letters": [
+    /// #                 "d",
+    /// #                 "e",
+    /// #                 "f",
+    /// #             ]
+    /// #         }
+    /// #     "#;
+    /// #     diesel::insert_into(d_e_f).values(from_str(d_e_f_data));
```

Comment:
Use the [`json!`](https://docs.serde.rs/serde_json/macro.json.html) macro + batch inserts here. This should simplify the code quite a lot and fix the errors thrown by CI.